README.md

```diff
@@ -35,6 +35,7 @@ HealthChecks packages include health checks for:
 - Uri: single uri and uri groups
 - Consul
 - Hangfire
+- RavenDB
```

Comment:
I think it would be better adding RavenBD close to the other database healthchecks (mysql, oracle, sqlite, postgres)

Reply:
I will do, Thanks.